src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestV4ServiceImpl.java

```diff
@@ -186,13 +186,15 @@ protected ConcurEventNotificationProcessingResultsDTO processTravelRequest(Strin
         }
         
         String reportNumber = requestAsListItem.getRequestId();
-        String travelerName = String.join(StringUtils.SPACE, requestAsListItem.getOwner().getFirstName(), 
+        String reportName = requestAsListItem.getName();
+        String reportStatus = requestAsListItem.getApprovalStatus() != null   ? requestAsListItem.getApprovalStatus().getName() : CUKFSConstants.NULL;
```

Comment:
Rather than printing "NULL" I think an empty string would make more sense to Tami